src/pothole_detection/Potholes/split.py

```diff
@@ -2,10 +2,10 @@
 from pathlib import Path
 from random import randint
 
-TRAIN_NUM = 1635
-VAL_NUM = 467
-TEST_NUM = 233
-TOTAL = 2335 - 1
+TRAIN_NUM = 3239
```

Comment:
Maybe a comment on where these numbers come from

Reply:
Or if they're defined in relation to each other/the number of files, maybe you could compute this at runtime?